.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Drop this file

Reply:
Sorry, I shouldn't have run `git add . `.
Updated.